src/components/Card/Card.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "../Card/Card.css";
+
+function Card({ children, cardImage, title, text, website }) {
+    const currentDate = new Date();
+    const formattedDate = currentDate.toLocaleDateString('en-US', {
+        year: 'numeric',
+        month: 'long',
+        day: 'numeric'
+    })
+
+    return (
+        <div className="card">
+            {children}
+            <img src={cardImage} className="card__image " alt="news article card" />
```

Comment:
`alt="news article card" `

Every tag `img` should have an informative attribute `alt` (the **name** of the image is usually there) to show the `alternative` text if the image is not uploaded and it’s needed for **screen readers**


` alt={title}`

Reply:
Done